queue/init.lua

```diff
@@ -24,28 +24,85 @@ end
 queue = setmetatable({
     driver = core_drivers,
     register_driver = register_driver,
-}, { __index = function() print(debug.traceback()) error("Please run box.cfg{} first") end })
+}, { __index = function()
+        print(debug.traceback())
```

Comment:
Hm. Hmmm. Where the traceback will appear in a real application? AFAIR, it depends on whether box.cfg() was called and so whether a log was configured. I would not use `print()` in a module, it is stated as the function purely for debug purposes, AFAIR.

Reply:
I see, it was here. Nevermind.